CrewChiefV4/R3E/R3EGameStateMapper.cs

```diff
@@ -549,6 +549,14 @@ public GameStateData mapToGameStateData(Object memoryMappedFileStruct, GameState
                     {
                         if (participantStruct.TrackSector == 1)
                         {
+                            // if the player's sector time is invalidated the game won't update SectorTimePreviousSelf so if the reported time is identical
+                            // to the previous time assume it's an invalid sector:
+                            if (currentGameState.SessionData.CurrentLapIsValid && 
+                                participantStruct.SectorTimePreviousSelf.Sector3 == currentGameState.SessionData.LastSector3Time)
+                            {
+                                currentGameState.SessionData.CurrentLapIsValid = false;
```

Comment:
Wouldn't it be helpful to have some tracing here, to understand how often this happens, and possibly, under which conditions?

Reply:
Good point. I ran a trace I'm debug and found the opponent data check triggering often (I wonder if there's been a regression in the game's shared memory here). The player one didn't trigger at all so I'll take this out